src/types/PlantingSite.ts

```diff
@@ -15,3 +17,5 @@ export type PlantingSiteZone = {
   name: string;
   plantingSubzones: PlantingSiteSubzone[];
 };
+
+export type PlantingSiteReportedPlants = components['schemas']['PlantingSiteReportedPlantsPayload'];
```

Comment:
rename this to `PlantingSiteReportedPlantsPayload` and add another type extending this with totalSeedlingsSent?

PlantingSiteReportedPlants = PlantingSiteReportedPlantsPayload & totalSeedslingsSent

Reply:
Ok I'm sorry my point was missed. My point was to create a type for what we are returning. Figured that would be clear.